.browserslistrc

```diff
@@ -1,4 +1,6 @@
-# Use the default value for browsers we support https://browsersl.ist/#q=defaults
-# Used by post-css only, JS browser targeting is controlled by Vite https://vitejs.dev/config/build-options.html#build-target
```

Comment:
Take a look at this comment you removed (about how this is just for post-css...). We need to wire up Vite to use `.browserslistrc` in place of `build.target` using the `browserslist-to-esbuild` module it looks like: https://github.com/vitejs/vite/issues/11489
A blog post about Vite and browserslist may be information as well: https://dev.to/meduzen/when-vite-ignores-your-browserslist-configuration-3hoe

Otherwise this `.browserslistrc` is only going to be used for CSS, and not used when building JS.

We should also be able to compare the package size before/after and see the impact (check the size of the `code-studio/build` folder after doing an `npm run build` before and after).